lib/util/cmcd_manager.js

```diff
@@ -75,9 +75,9 @@ shaka.util.CmcdManager = class {
     this.startTimeOfLoad_ = 0;
 
     /**
-     * @private {boolean}
+     * @private {{request: boolean, response: boolean}}
      */
-    this.msdSent_ = false;
+    this.msdSent_ = {request: false, response: false};
```

Comment:
Esto es solo por legibilidad
```suggestion
    this.msdSent_ = {
      request: false, 
      response: false,
    };
```